xml/vk.xml

```diff
@@ -13769,10 +13781,13 @@ typedef void <name>CAMetalLayer</name>;
                 <enum value="&quot;VK_EXT_extension_340&quot;"              name="VK_EXT_EXTENSION_340_EXTENSION_NAME"/>
             </require>
         </extension>
-        <extension name="VK_EXT_extension_341" number="341" author="EXT" contact="Joshua Ashton @Joshua-Ashton" supported="disabled">
+        <extension name="VK_VALVE_partial_present" number="341" type="device" requires="VK_KHR_incremental_present" author="VALVE" contact="Joshua Ashton @Joshua-Ashton" supported="vulkan">
             <require>
-                <enum value="0"                                             name="VK_EXT_EXTENSION_341_SPEC_VERSION"/>
-                <enum value="&quot;VK_EXT_extension_341&quot;"              name="VK_EXT_EXTENSION_341_EXTENSION_NAME"/>
+                <enum value="1"                                             name="VK_VALVE_PARTIAL_PRESENTATION_SPEC_VERSION"/>
+                <enum value="&quot;VK_VALVE_partial_present&quot;"     name="VK_VALVE_PARTIAL_PRESENTATION_EXTENSION_NAME"/>
+                <enum offset="0" extends="VkStructureType"                  name="VK_STRUCTURE_TYPE_PRESENT_REGIONS2_VALVE"/>
+                <type name="VkPresentRegions2VALVE"/>
+                <type name="VkUpdateRegionVALVE"/>
             </require>
```

Comment:
Should this be dependent on `VK_KHR_incremental_present`?

I guess the behavior outlines in this extension implies support for incremental present, given that incremental present is basically "the driver **may choose to only** present in these regions", while partial present is "the driver **must only** present in these regions".